indexing-service/src/main/java/org/apache/druid/indexing/common/task/AppenderatorDriverRealtimeIndexTask.java

```diff
@@ -316,7 +316,10 @@ public TaskStatus run(final TaskToolbox toolbox)
             AppenderatorDriverAddResult addResult = driver.add(inputRow, sequenceName, committerSupplier);
 
             if (addResult.isOk()) {
-              if (addResult.isPushRequired(tuningConfig)) {
+              final boolean isPushRequired =
```

Comment:
Any reason not to push this down into the `AppenderatorDriverAddResult` like it was previously? Is the other `AppenderatorDriverAddResult.isPushRequired` method still legitimately used?

Reply:
Hmm, yeah it looks better to use the existing one. Reverted to use it.